doc/content/getting_started/contributing.md

```diff
@@ -0,0 +1,244 @@
+your changes. In any case, you will need to list an issue number in one of your commits! To get started:
+
+1. Select “New issue” on this page: [https://github.com/idaholab/tmap8/issues](https://github.com/idaholab/tmap8/issues)
+1. Select either “bug report” or “feature request” (most should be feature requests)
+1. Provide issue descriptions of a couple of sentences (or as much as you see fit) following the prompts.
+1. Create issue
+
+You will now have an issue number next to the title. We will use it to link the pull request (PR) to the issue.
+
+## Work in a GitHub fork
+
+!style halign=left
+TMAP8 development follows a "Fork & Pull" development process. See more information on
+[forking repositories in the GitHub documentation](https://guides.github.com/activities/forking).
+
+### Fork TMAP8 and clone
+
+1. Navigate to [the TMAP8 repository](https://github.com/idaholab/tmap8)
+1. Click the "Fork" button on the upper right of the code file tree to have a copy of TMAP8 in your own account
+1. Clone your new fork to your local machine using the following command (this assumes you
+   [have SSH set up on your GitHub account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account))):
+
+   ```
+   cd ~/projects
+   git clone git@github.com:<your_user_name>/tmap8.git
+   ```
+
+   In your local copy, this clone URL is given the shorthand name "origin". This will become relevant
+   later in this document.
+
+### Add an upstream git remote
+
+!style halign=left
+Add the main TMAP8 repository as an "upstream" git remote, so that you can
+[git fetch](https://git-scm.org/docs/git-fetch) updates from the main repository:
+
+```
+cd ~/projects/tmap8
+git remote add upstream git@github.com:idaholab/tmap8.git
+```
+
+To fetch changes, run:
+
+```
+git fetch upstream
+```
+
+## Creating a new branch
+
+Before make any changes locally (which would require having a working version of the application),
+we will need to create a new branch.
+
+1. In your terminal, go to your application:
+
+   ```
+   cd ~/projects/tmap8
+   ```
+
+1. It is usually best to make sure your changes are based on devel:
+
+   ```
+   git checkout devel
+   ```
+
+1. Clean and reset devel to get it up to date (make sure you do not have any changes on devel, because you would lose that):
+
+   ```
+   git reset --hard upstream/devel && git submodule update --init
+   ```
+
+1. Create new branch:
+
+   ```
+   git checkout -b <BRANCH_NAME>
+   ```
+
+## Make modifications and commit (the regular development workflow)
+
+At this point, you have a local branch that is up to date with the upstream remote. You’re now ready
+to make your changes. Remember, **small, consistent changes** are much better than dumping massive,
+unrelated changes all at once.
+
+1. Make your changes locally to address the issue (partly or fully) - using
+   [VSCode is recommended](help/development/VSCode.md).
```

Comment:
This page needs to be added to the content section of `config.yml` under `modules:`